### PR TITLE
Add script to generate notebooks

### PR DESCRIPTION
**Pull request by yao-cqc**
This should make script generation much easier.

### CODE REVIEW

File: examples/check-examples [hunk 1]
```diff
@@ -2,12 +2,13 @@
 
 set -e
 
-for file in $(ls python/*py | xargs -n 1 basename)
+for name in `cat maintained-notebooks.txt`
```
Comment (ferbetanzo) [inline examples/check-examples:5]: Do you think we would have notebooks that are generated with python but that we'd like to skip in the `check-examples` and `gen-nb` scripts? If not, I think getting all the scripts under python/ is better.

Comment (yao-cqc) [reply examples/check-examples:5]: Yeah, I think we should check all notebooks at least for now. Do you think we should remove ``maintained-notebooks.txt`` all together? If we don't keep unchecked notebooks then I'm not sure what is the point of this file. We can always bring this file back when needed.

Comment (ferbetanzo) [reply examples/check-examples:5]: Yeah, I think removing `maintained-notebooks.txt` is good :)

**Review by ferbetanzo — APPROVED** [review]
Looks great, thx!

**Pull request merged**